packages/fluent-editor/src/custom-uploader.ts

```diff
@@ -97,14 +97,16 @@ class CustomUploader extends Uploader {
 
   // 处理上传文件
   handleUploadFile(range, files, _hasRejectedFile) {
+    const fileUrl = URL.createObjectURL(files[0])
     this.insertFileToEditor(range, files[0], {
       code: 0,
       data: {
         title: files[0].name,
         size: files[0].size,
-        src: files[0].src,
+        src: fileUrl,
```

Comment:
如果用户上传文件到服务器，会有 src，src 优先级高。如果有 src，以 src 为准；如果没有 src，则使用 fileUrl。
```suggestion
        src: files[0].src ?? fileUrl,
```

Reply:
好的